modules/create-object-collections.adoc

```diff
@@ -0,0 +1,66 @@
+.. Click *+Attach*. Collections you attach are listed in the *Attached collections* section.
+. Click *Save*. 
+
+[id="creating-collection-rules"]
+== Creating collection rules
+
+Use the following guidelines when creating collection rules:
+
+* You must configure at least one rule or attach another collection to the new one that you are creating. 
++
+[NOTE]
+====
+For {product-title-short} release 3.74, collections are supported only for deployments, so you must configure a deployment object.
+====
+*  You first configure rules to include all or specific deployments, and then you can further filter the collection to include objects meeting certain criteria in namespaces and clusters.
+
+.Procedure
+
+. In the *Deployments* section, choose one of the following options from the drop-down list:
+
+* *All deployments*: Includes all deployments in the collection. If you choose this option, you must then filter the collection using namespaces or clusters.
+* *Deployments with names matching* or *Deployments with labels matching*: Click one of these options to select by name or label and then click one of the following options:
+** Select *An exact value of* and enter the exact name of the object.
+** Select *A regex value of* to use regular expression, also called "regex," to search for an object. This option is useful if you do not know the exact name of the object. A regular expression is a string of letters, numbers, and symbols that defines a pattern. {product-title-short} uses this pattern to match characters or groups of characters and return results. For more information about regular expression, see "Regular-Expressions.info" in the "Additional resources" section.
+. Optional: To add more name matching criteria, click *OR* and configure another exact or regular expression value.
+
+The following example provides the steps for configuring a collection for a medical application. In this example, you want your collection to include the `reporting` deployment, a database called `patient-db`, and you know that the database is located in a namespace with a label of `medical`. You would perform the following steps:
+
+. In *Collection rules*, select *Deployments with names matching*.
+. Click *An exact value of* and enter *reporting*.
+. Click *OR*.
+. Click *A regex value of* and enter `.*-db` to select all databases. The panel on the right might display databases that you do not want to include. You can exclude those databases by using additional filters. For example:
+.. Filter by namespace labels by clicking *Namespaces with labels matching exactly* and entering `medical` to include only databases in the namespace that is labeled medical.
```

Comment:
+1 
Also, the doc does not mention anywhere that labels should be of the form `key=value` and would be treated as exact match. So it would be worth mentioning that somewhere in the procedure to create collection section, in addition to the example here.